tests/units/units.cpp

```diff
@@ -665,6 +665,29 @@ TEST(Units, compareMultiplierStandardUnit)
     EXPECT_NEAR(500.0, libcellml::Units::scalingFactor(u2, u1), 1e-12);
 }
 
+TEST(Units, compareMultiplierFirstParameterNullUnit)
+{
+    libcellml::UnitsPtr u = libcellml::Units::create();
+    u->setName("u");
+    u->addUnit(libcellml::Units::StandardUnit::LUX, 0, 1.0, 1.0);
+
+    EXPECT_EQ(0.0, libcellml::Units::scalingFactor(nullptr, u));
+}
+
+TEST(Units, compareMultiplierSecondParameterNullUnit)
```

Comment:
Similarly, rename the test `compareScalingFactorWithNullptrAsSecondParameter`?

Reply:
Yep, I'll implement that now.